pkg/collector/corechecks/ebpf/oom_kill.go

```diff
@@ -114,10 +113,14 @@ func (m *OOMKillCheck) Run() error {
 			continue
 		}
 		entityID := containers.BuildTaggerEntityName(line.ContainerID)
-		tags, err := tagger.Tag(entityID, collectors.OrchestratorCardinality)
-		if err != nil {
-			log.Errorf("Could not collect tags for container %s: %s", line.ContainerID, err)
+		var tags []string
```

Comment:
Nit: code seems identical - good opportunity to extract into a helper and reuse?